onnxruntime/core/providers/dnnl/subgraph/dnnl_layernorm.cc

```diff
@@ -174,10 +172,8 @@ void DnnlLayerNorm::CreatePrimitive(DnnlSubgraphPrimitive& sp, DnnlNode& node) {
 
   // Get epsilon to avoid zero division
   float epsilon = GetEpsilon(node);
-  // Operation desciptor
-  auto lnorm_desc = dnnl::layer_normalization_forward::desc(prop_kind, src_md, epsilon, op_flags);
   // Primitive desciptor
-  auto lnorm_pd = dnnl::layer_normalization_forward::primitive_desc(lnorm_desc, dnnl_engine);
+  auto lnorm_pd = dnnl::layer_normalization_forward::primitive_desc(dnnl_engine, prop_kind, src_md, src_md, epsilon, op_flags);
```

Comment:
in the original layer_normalization desc src_md is only listed one time yet here is it listed twice.  Is that correct?

Reply:
Yes, previously most of the primitive descriptors didn't require a dst_md to generate the descriptor, now all primitive descriptors require a memory descriptor for each input and output they have, since this primitive is going to be executed in place the src_md is the same as the dst_md